saleor/data_feeds/google_merchant.py

```diff
@@ -0,0 +1,219 @@
+from __future__ import unicode_literals
+
+import gzip
+import csv
+from os import path
+
+from django.conf import settings
+from django.contrib.sites.models import Site
+from django.contrib.syndication.views import add_domain
+from django.core.files.storage import default_storage
+
+from ..discount.models import Sale
+from ..product.models import ProductVariant, Category
+
+CATEGORY_SEPARATOR = ' > '
+
+FILE_PATH = path.join(settings.INTEGRATIONS_DIR, 'saleor-feed.csv.gz')
+FILE_URL = default_storage.url(FILE_PATH)
+COMPRESSION = False
+
+ATTRIBUTES = ['id', 'title', 'product_type', 'google_product_category',
+              'link', 'image_link', 'condition', 'availability',
+              'price', 'tax', 'shipping', 'sale_price',
+              'mpn', 'brand', 'item_group_id', 'gender', 'age_group',
+              'color', 'size', 'description']
+
+
+def get_feed_items():
+    items = ProductVariant.objects.all()
+    items = items.select_related('product')
+    items = items.prefetch_related(
+        'images', 'stock', 'product__attributes', 'product__categories',
+        'product__images',
+    )
```

Comment:
Style: please drop the hanging comma and close the parenthesis at the end of the previous line.